examples/platforms/nrf528xx/nrf52840/openthread-core-nrf52840-config.h

```diff
@@ -279,24 +279,25 @@
 #endif
 
 /**
- * @def OPENTHREAD_CONFIG_CSL_SAMPLE_WINDOW
+ * @def OPENTHREAD_CONFIG_CSL_RECEIVE_TIME_AHEAD
  *
- * CSL sample window in units of 10 symbols.
+ * Reception scheduling and ramp up time needed for the receiver to be ready, in units of 10 symbols.
```

Comment:
This was just keeping previous code base, which I guess it was based on the IEEE 802.15.4 CSL Period definition, also in units of 10 symbols. But I agree there is no reason not to have better granularity here. It would also save some multiplications in the code.